config.go

```
@@ -112,6 +114,31 @@ func (c *config) reload() error {
 	if err := yaml.Unmarshal(by, &cfg); err != nil {
 		return fmt.Errorf("parse config: %w", err)
 	}
+	// if serverConfigPath is defined we also read all y(a)ml files from that directory into the serviceConfigs
+	if cfg.ServerConfigPath != "" {
+		files, err := os.ReadDir(cfg.ServerConfigPath)
+		if err != nil {
+			log.Printf("WARN: configured ServerConfigPath \"%s\" couldn't be loaded", cfg.ServerConfigPath)
```

I think this should return an error, rather than just logging a warning.